### ISSUE TITLE
Contrast Issue of tags

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#474]
## Problem

![84492434-cdd35200-acc3-11ea-98df-800114727e44](https://user-images.githubusercontent.com/45792768/84812709-723ef680-b02c-11ea-8e79-95215dcab65c.png)
![84492451-d3309c80-acc3-11ea-8155-8f66c648f787](https://user-images.githubusercontent.com/45792768/84813199-2fc9e980-b02d-11ea-9c1a-9895f2b9b221.png)

## Description

Currently, this span has a contrast ratio of **4.42(both with and without hover)** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @dhruvkb

**Comment by zackkrida** [comment]
I would recommend we change $color-dark-gray: https://github.com/creativecommons/vocabulary/blob/master/src/styles/color.scss#L11

from `rgb(120,120,120)` to something like `rgb(119, 118, 117)`. There's a barely perceptible difference but it results in a passing score to slightly de-saturate the color. Thoughts @panchovm?

**Comment by fcoveram** [comment]
You are right @zackkrida. It is a slight visual change but a huge one for Figma components, but it worths.

I was trying options and we can use `rgb(118, 118, 118)`, `#767676` in hex code. I will update the color and all components will update automatically.

**Comment by fcoveram** [comment]
Done. I just created [this ticket](https://github.com/creativecommons/vocabulary/issues/527) for this.

**Comment by ayanchoudhary** [comment]
The change looks good, we can pass the accessibility requirements without changing the color scheme too much.

**Comment by zackkrida** [comment]
Closing as this was fixed!
